### PR TITLE
feat: adding samples to start/stop/reset operations

### PR DESCRIPTION
**Pull request by m-strzelczyk**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)

b/192839790

Will be added to [this page](https://cloud.google.com/compute/docs/instances/stop-start-instance)

### CODE REVIEW

**Comment by m-strzelczyk** [comment]
@engelke can you have a look?

**Comment by m-strzelczyk** [comment]
The tests should be finally fixed now. @engelke can  you review?

File: samples/snippets/test_sample_default_values.py [hunk 1]
```diff
@@ -37,13 +38,15 @@ def temp_bucket():
 def test_set_usage_export_bucket_default(capsys: typing.Any,
                                          temp_bucket: storage.Bucket) -> None:
     set_usage_export_bucket(project_id=PROJECT, bucket_name=temp_bucket.name)
+    time.sleep(5)  # To make sure the settings are properly updated
```
Comment (engelke) [inline samples/snippets/test_sample_default_values.py:41]: Is this necessary? set_usage_export_bucket waits for the operation to complete before returning. Is there a potential additional wait for the state to replicate through cloud storage?

**Review by engelke — APPROVED** [review]
LGTM

File: samples/snippets/test_sample_default_values.py (hunk 1 above)
Comment (m-strzelczyk) [reply samples/snippets/test_sample_default_values.py:41]: Yes, this is necessary. Without this, the script sometimes grabbed previous values - the change propagation wasn't fast enough.

**Pull request merged**